src/modules/addcomment.js

```diff
@@ -0,0 +1,10 @@
+const addcomment = (Obj) => fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/1RjuQJrVTvT45Mcm7X6n/comments', {
```

Comment:
Good job team! but it's highly recommended to not store confidential information about the API in your repository. So please make sure to remove the API key from here. And you can update your Readme for proper installation guide as well.

Reply:
I've implemented it. Thank you for replying to me on Slack and teaching me new things. Also, letting me know about a new term, ".env". I'll surely put some effort in learning that whenever I managed to squeeze time out of Microverse.